hack/components/bump-multus-dynamic-networks.sh

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env bash
+
+set -xeo pipefail
+
+source hack/components/yaml-utils.sh
+source hack/components/git-utils.sh
+source hack/components/docker-utils.sh
+
+function __parametize_by_object() {
+  for f in ./*; do
+    case "${f}" in
+      ./ConfigMap_dynamic-networks-controller-config.yaml)
+        yaml-utils::update_param ${f} metadata.namespace '{{ .Namespace }}'
+        yaml-utils::remove_single_quotes_from_yaml ${f}
+        ;;
+      ./ClusterRoleBinding_dynamic-networks-controller.yaml)
+        yaml-utils::update_param ${f} subjects[0].namespace '{{ .Namespace }}'
+        yaml-utils::remove_single_quotes_from_yaml ${f}
+        ;;
+      ./DaemonSet_dynamic-networks-controller-ds.yaml)
+        yaml-utils::update_param ${f} metadata.namespace '{{ .Namespace }}'
+        yaml-utils::set_param ${f} spec.template.spec.containers[0].imagePullPolicy '{{ .ImagePullPolicy }}'
+        yaml-utils::update_param ${f} spec.template.spec.containers[0].image  '{{ .MultusDynamicNetworksControllerImage }}'
+        yaml-utils::set_param ${f} spec.template.spec.affinity '{{ toYaml .Placement.Affinity | nindent 8 }}'
```

Comment:
Is it deliberately that there isn't `nodeSelector` ?

Is it deliberately that the tesst "and workload PlacementConfiguration is deployed on components"
doesn't include it ?